src/index.ts

```diff
@@ -151,6 +156,37 @@ function removeOperations<T extends Operation>(
   }
 }
 
+function redirectPaths(acceptedOp: OperationMove, proposedOps: Operation[]): void {
+  const acceptedFrom = acceptedOp.from;
+  const acceptedPath = acceptedOp.path;
+
+  for (const proposedOp of proposedOps) {
+    const matchesPathToFrom = proposedOp.path === acceptedFrom || proposedOp.path.indexOf(acceptedFrom + '/') === 0;
+
+    let matchesFromToFrom = false;
+    if (isFromOperation(proposedOp)) {
+      matchesFromToFrom = proposedOp.from === acceptedFrom || proposedOp.from.indexOf(acceptedFrom + '/') === 0;
+    }
+
+    const isProposedOpFrom = isFromOperation(proposedOp);
+
+    if (!isProposedOpFrom && matchesPathToFrom) {
+      proposedOp.path = acceptedPath + proposedOp.path.substr(acceptedFrom.length);
+      proposedOp.redirected = true;
+    } else if (isProposedOpFrom && matchesFromToFrom) {
+      const proposedFromOp = proposedOp as FromOperation;
+      proposedFromOp.from = acceptedPath + proposedFromOp.from.substr(acceptedFrom.length);
+      proposedFromOp.redirected = true;
+    }
+  }
```

Comment:
What happens if `isProposedOpFrom && matchesPathToFrom`? E.g. proposed op is `{op: OpType.move, from: '/array/4', path: '/array/6'},` and accepted op is `{op: OpType.move, from: '/array/6', path: '/array/0'}`

Reply:
It shifts the indices of the proposed op so the result is:
`{"from": "/array/5", "op": "move", "path": "/array/7"}`
I assume that's not correct, I'm not sure what the intended behavior should be though

Reply:
Hm.. I guess that's fine, the `from` is transformed correctly. Ok, how about nested change: 
accepted: `{op: OpType.move, from: '/rows/6', path: '/rows/0'}`
proposed: `{op: OpType.move, from: '/rows/4/columns/0/modules/2', path: '/rows/6/columns/0/modules/1'}`? That should be transformed to `{op: OpType.move, from: '/rows/5/columns/0/modules/2', path: '/rows/0/columns/0/modules/1'}`

Reply:
You're right, that wasn't working but it should be fixed now. I've also added this case and inverted one (moving from index 0 to 6) to the tests.